pages/editor.vue

```diff
@@ -85,6 +86,23 @@
         });
     }
 
+    function handleGoingBack() {
+        if (userDidSomething.value) {
+            goingBackPopUpShow.value = true;
+        } else {
+            return reloadNuxtApp({
+                path: '/',
+                ttl: 1000,
+            });
+        }
+    }
+
+    window.onbeforeunload = function (e) {
+        if (userDidSomething.value && e.target.activeElement === this.document.body) {
```

Comment:
It is not, calling the reloadNuxtApp also trigger before unload. So when you use our button it would show the popup. since we have no call on body and that the browser reloard is attach to it this check is very useful (only trigger the browser reload).

Reply:
It is not, calling the reloadNuxtApp also trigger before unload. So when you use our button it would show the popup. since we have no call on body and that the browser reloard is attach to it this check is very useful (only trigger the browser reload).